README.md

```diff
@@ -1 +1,119 @@
-# check-status 
+<!-- start title -->
+
+# gitgitgadget Action: check-status
+
+<!-- end title -->
+<!-- start description -->
+
+This is a GitHub action to perform various scheduled actions with projects using gitgitgadget to submit changes. It is used as a scheduled action running in a separate repo or the repo it is acting upon. The use case is for repos that will not allow complex GitHub actions in the codebase when the GitHub repo is a clone that is used to submit updates to a non-GitHub maintained repo.
+
+<!-- end description -->
+<!-- start contents -->
+<!-- end contents -->
+
+## Usage
+
+<!-- start usage -->
+
+```yaml
+- uses: gitgitgadget/check-status@v0.5.0
+  with:
+    # The action to be performed. It must be one of the following: "update-open-prs",
+    # "update-commit-mappings", "handle-open-prs", "handle-new-mails".
+    action: ""
+
+    # Repository owner.
+    repo-owner: ""
```

Comment:
Should we default to `${{ github.repository_owner }}`?

Reply:
This would be running on a proxy repo (gitgitgadget-workflows?) so that value would not be valid at this point.

Reply:
I meant for projects that add this workflow directly into their default branch. But that may never be the common case.

Reply:
> I meant for projects that add this workflow directly into their default branch. But that may never be the common case.

In that case, can't they specify the variable in the workflow?  Allowing a default that is incorrect for many use cases could lead to confusion (and issues).

Reply:
True.